Sming/SmingCore/Platform/OSTimer.h

```diff
@@ -0,0 +1,101 @@
+/****
```

Comment:
You have SimpleTimer declared two times. Once `Sming/SmingCore/Platform/OSTimer.h` and a second time in `Sming/SmingCore/SimpleTimer.h`. Maybe you should remove `Sming/SmingCore/Platform/OSTimer.h`.

Reply:
Finger trouble, fixed.